hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -177,7 +177,6 @@ def test_metadata(self):
         np.testing.assert_approx_equal(
             TEM_md.Detector.EDS.energy_resolution_MnKa, 130)
 
-        self.signal.set_microscope_parameters(real_time=3.1)
```

Comment:
Either this is no longer needed and the implementation is removed from the code or it needs to be tested. I would suggest to keep it.